Wikipedia/Code/WKWebView+WMFWebViewControllerJavascript.swift

```diff
@@ -142,10 +142,7 @@ extension WKWebView {
     }
     
     private func languageJS(for article: MWKArticle) -> String {
-        guard let lang = (article.url as NSURL).wmf_language else {
-            assertionFailure("Expected lang")
-            return ""
-        }
+        let lang = (article.url as NSURL).wmf_language ?? MWKLanguageLinkController.sharedInstance().appLanguage?.languageCode ?? "en"
```

Comment:
Are we sure we want `appLanguage` here? The assertion seemed to be useful to know asap that we're getting what we expect...

Reply:
@montehurd appLangugae then english seemed like a reasonable fallback path for sites without language. The help page is https://m.mediawiki.org/wiki/Wikimedia_Apps/iOS_FAQ/

Reply:
@montehurd look at the commits after d563e42 (if the diff is confusing)

Reply:
@joewalsh defaulting to EN is fine but it's no guarantee the title on EN matches whatever that article's title is in device lang. Just suspecting the assert is useful for early warning in case something ever goes crazy wrong...

Reply:
@montehurd feel free to submit an alternate fix or re-add the assertion without the guard/return. falling back on default languages seemed like the lowest impact fix for the help & feedback page for 5.7.3.